test/Datadog.Trace.TestHelpers/MockTracerAgent.cs

```diff
@@ -272,6 +272,11 @@ private void HandleHttpRequests()
                     ctx.Response.OutputStream.Write(buffer, 0, buffer.Length);
                     ctx.Response.Close();
                 }
+                catch (InvalidOperationException)
```

Comment:
CI is complaining about this.

Reply:
Sorry about that, fixed in 9ec03177d40cb31fb12f475e9f17580d42a7d1f3